spinalcordtoolbox/process_seg.py

```diff
@@ -231,3 +233,127 @@ def _find_AP_and_RL_diameter(major_axis, minor_axis, orientation, dim):
     diameter_AP *= dim[0]
     diameter_RL *= dim[1]
     return diameter_AP, diameter_RL
+
+
+def get_angle_correction(im_seg):
```

Comment:
a very similar code is already in process_seg-- could you please try to see if we can replace the current code with your function here?

Reply:
Done :)